### DISCUSSION TITLE
Error opening ExodusII mesh file

### DISCUSSION THREAD

**Discussion by amosaha** [idaholab/moose#19264] [Q&A Meshing] [answered]
Hello,
The mesh is made by trelis and can be opened by the peacockand. I have updated all,but there's an error I can't solve.
This is my input file :
```
[Mesh]
  [fmesh]
    type=FileMeshGenerator
    file =hollowcylinder_and_cylinder_merge.e
  []
[]
```
and the error :

Error opening ExodusII mesh file: /home/amos/projects/cat/hollowcylinder_and_cylinder_merge.e

Thanks very much，I hope your help.

**Comment by GiudGiud** [comment, marked as answer]
Hello

I cant think of what would be the problem. Would you mind sharing that mesh file with us?
Otherwise, can you please double check the spelling of the file name in the input. 

Guillaume

**Reply by amosaha** [reply]
Hello Guillaume
I have checked my spelling of the file name,and maybe the problem is the mesh file.
This is my mesh file.
I'm looking forward your suggestion,Thanks.

[hollowcylinder_and_cylinder_merge.zip](https://github.com/idaholab/moose/files/7447895/hollowcylinder_and_cylinder_merge.zip)

**Reply by GiudGiud** [reply]
I'm getting the error below with your file.
Could this be that this is an exodus 8 file?
I m going to defer to @roystgnr who has a lot of expertise on exodus.

```
EXODUS: Error: Attempting to open the netcdf-4 file:
	'/home/guillaume/projects/moose/test/tests/meshgenerators/file_mesh_generator/hollowcylinder_and_cylinder_merge.e'
	with a netcdf library that does not support netcdf-4
Error opening ExodusII mesh file: /home/guillaume/projects/moose/test/tests/meshgenerators/file_mesh_generator/hollowcylinder_and_cylinder_merge.e
Stack frames: 15
0: libMesh::print_trace(std::ostream&)
1: libMesh::MacroFunctions::report_error(char const*, int, char const*, char const*)
2: libMesh::ExodusII_IO_Helper::open(char const*, bool)
3: libMesh::ExodusII_IO::read(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
4: FileMeshGenerator::generate()
5: MeshGenerator::generateInternal()
6: MooseApp::executeMeshGenerators()
7: Action::timedAct()
8: ActionWarehouse::executeActionsWithAction(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
9: ActionWarehouse::executeAllActions()
10: MooseApp::runInputFile()
11: MooseApp::run()
12: ../../..//moose_test-opt(+0x3153) [0x55d262d4a153]
13: __libc_start_main
14: ../../..//moose_test-opt(+0x33dd) [0x55d262d4a3dd]
[0] ../src/mesh/exodusII_io_helper.C, line 551, compiled Sep 14 2021 at 00:15:56
```

**Reply by amosaha** [reply]
I made this mesh file by Trelis 16.5. And this is the output format.
<img width="153" alt="pic 1" src="https://user-images.githubusercontent.com/93416273/139563991-bd563169-c113-4ed8-9cd9-30b9949096e5.png">

**Reply by GiudGiud** [reply]
can you try using the normal format?

**Reply by amosaha** [reply]
oh,thanks,moose is working well now.

**Reply by roystgnr** [reply]
Sorry to have missed this.  libMesh can be built to support HDF5 ExodusII files too, but it takes explicit configure options to do so and Moose's build script doesn't yet add those by default.

**Reply by amosaha** [reply]
hello,I want to use a file of HDF5 ,how can make the configuration?
